aiodogstatsd/client.py

```diff
@@ -33,6 +34,7 @@ def __init__(
         constant_tags: Optional[typedefs.MTags] = None,
         read_timeout: float = 0.5,
         close_timeout: Optional[float] = None,
+        sample_rate: Optional[typedefs.MSampleRate] = 1,
```

Comment:
It can't be optional, the type should be just `typedefs.MSampleRate`, because if you pass `None` here then code will fail because of:
```
sample_rate = sample_rate or self._sample_rate
if sample_rate != 1 and random() > sample_rate:
    ...
```